en/docs/develop/product-apis/publisher-apis/publisher-v1/publisher-v1.yaml

```diff
@@ -7870,6 +8236,7 @@ definitions:
         type: array
         items:
           type: string
+        example: []
```

Comment:
Can't we have actual examples here? Does it break any intended behaviors of the doc?

Reply:
this field is used when cross tenant subscription is enabled. Since we are deprecating that feature in 3.2.0 left the field as an empty array.